index.html

```diff
@@ -1918,6 +1974,12 @@
           <h2>
             <dfn data-lt="updateWith(detailsPromise)">updateWith()</dfn> method
           </h2>
+          <p class="note">
+            If the web page wishes to update the payment request then it can
+            call <a>updateWith()</a> and provide a promise that will resolve
+            with a <a>PaymentDetailsUpdate</a> dictionary containing changed
```

Comment:
Maybe point out that you don't have to provide a promise. E.g. "provide a PaymentDetailsUpdate dictionary, or a promise for one, containing..."